app/router.go

```diff
@@ -129,7 +131,8 @@ var topologyRegistry = map[string]topologyView{
 }
 
 type topologyView struct {
-	human    string
-	parent   string
-	renderer render.Renderer
+	human       string
+	parent      string
+	renderer    render.Renderer
+	isMultiHost bool // does the view involve information from multiple hosts?
```

Comment:
Instead of this being set explicitly per view, can this be inferred from the node you've been asked about?  Ie examine all the origin ids for a given renderable node, fetch the node id from their node metadata, and see if you have more than one?